### DISCUSSION TITLE
Integration with CI/CD Github Action

### DISCUSSION THREAD

**Discussion by hieumt2** [cypress-io/cypress#14881] [CI setup] [answered]
How can I run cypress and check if any case failed, Github will throw an error and stop deploying?
I tried to start the server and `cypress run`. Despite many failed case, Github still shows that it's nice and continue deploying

**Comment by bahmutov** [comment, marked as answer]
I am not sure I understand the problem or how it can happen. If you execute Cypress on GitHub Actions, if there are any failed tests, it exits with non-zero exit code, and the action fails. Can you provide the output of the action run, or better link to the run itself showing it NOT to be the case?
